lib/url.js

```diff
@@ -827,8 +827,9 @@ Url.prototype.resolveObject = function resolveObject(relative) {
       //occasionally the auth can get stuck only in host
       //this especially happens in cases like
       //url.resolveObject('mailto:local1@domain1', 'local2@domain2')
-      const authInHost = result.host && result.host.indexOf('@') > 0 ?
-                       result.host.split('@') : false;
+      const authInHost =
+        result.host && result.host.indexOf('@') > 0 ?
+          result.host.split('@') : false;
```

Comment:
Good idea! I don't think there's much of a chance this is being done as a ternary because it's more performant than the alternative approach you're suggesting here. But that's usually behind my reluctance to alter anything other than whitespace in these things. :-D